hw_29.08.23/task4.py

```diff
@@ -0,0 +1,7 @@
+day = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
+
+number_of_the_week = {i + 1: day[i] for i in range(len(day))}
+day_of_the_week = {day[i]: i + 1 for i in range(len(day))}
```

Comment:
Also absolutely right, and again just as a simpler alternative you could have reversed previous dictionary